sdk/batch/azure-compute-batch/src/test/java/com/azure/compute/batch/SharedKeyTests.java

```diff
@@ -17,17 +17,15 @@
 import static java.time.OffsetDateTime.now;
 
 public class SharedKeyTests extends BatchServiceClientTestBase {
-    private static PoolClient poolClientWithSharedKey;
     private final String sharedKeyPoolId = "SharedKey-testpool";
     private final String vmSize = "STANDARD_D1_V2";
     private final String nodeAgentSkuId = "batch.node.ubuntu 18.04";
 
     @Override
     protected void beforeTest() {
         super.beforeTest();
-        BatchSharedKeyCredentials sharedKeyCred = getSharedKeyCredentials();
+        AzureNamedKeyCredential sharedKeyCred = getSharedKeyCredentials();
         batchClientBuilder.credential(sharedKeyCred);
-        poolClientWithSharedKey = batchClientBuilder.buildPoolClient();
```

Comment:
We'll still need to rebuild the client in order for the Shared Key Http Policy to take place.